src/sentry/static/sentry/app/components/dropdownControl.tsx

```diff
@@ -58,10 +75,10 @@ class DropdownControl extends React.Component {
       children,
       alwaysRenderMenu,
       alignRight,
-      menuOffset,
       menuWidth,
       blendWithActor,
     } = this.props;
+    const alignMenu = alignRight ? 'right' : 'left';
```

Comment:
alignMenu needs to be type `'left' | 'right'` not `string`. Assigning it to a variable instead of inline seems to cast to the right type.